src/pudl/metadata/fields.py

```diff
@@ -1970,6 +1982,15 @@
         "description": "The energy contained in fuel burned, measured in million BTU.",
         "unit": "MMBtu",
     },
+    "hour_of_year": {
+        "type": "number",
+        "description": "Number between 1 and 8670 representing the hour in a given year.",
+    },
+    "hour_utc": {
+        "type": "datetime",
+        "description": "Hourly datetime in UTC.",
```

Comment:
What do you mean by "hourly datetime" and "hour_utc"? If this is just a timestamp (2021-01-01 01:00:00), as compared to a date (2021-01-01), then "datetime_utc" column that already exists should work just fine instead of creating a new one.